docs/content/en/docs/configuration/command-line.md

```diff
@@ -36,11 +42,17 @@ The following command-line options are supported:
 | [`--publish-service`](#publish-service)                 | namespace/servicename      |                         |       |
 | [`--rate-limit-update`](#rate-limit-update)             | uploads per second (float) | `0.5`                   |       |
 | [`--reload-strategy`](#reload-strategy)                 | [native\|reusesocket]      | `reusesocket`           |       |
+| [`--report-node-internal-ip-address`](#report-node-internal-ip-address) | [true\|false] | `false`              |       |
 | [`--sort-backends`](#sort-backends)                     | [true\|false]              | `false`                 |       |
 | [`--sort-endpoints-by`](#sort-endpoints-by)             | [endpoint\|ip\|name\|random] | `endpoint`            | v0.11 |
 | [`--stats-collect-processing-period`](#stats)           | time                       | `500ms`                 | v0.10 |
+| [`--sync-period`](#sync-period)                         | time                       | `600s`                  |       |
 | [`--tcp-services-configmap`](#tcp-services-configmap)   | namespace/configmapname    | no tcp svc              |       |
+| [`--update-status`](#update-status)                     | [true\|false]              | `true`                  |       |
+| [`--update-status-on-shutdown`](#update-status-on-shutdown) | [true\|false]          | `true`                  |       |
+| [`--validate-config`](#validate-config)                 | [true\|false]              | `false`                 |       |
 | [`--verify-hostname`](#verify-hostname)                 | [true\|false]              | `true`                  |       |
+| [`--version`](#version)                                 | [true\|false]              | `false`                 |       |
```

Comment:
`--version` was missing?

Reply:
Seems it was! I mean, it's not very important either.  But with these things i guess i naturally err on the side of preferring to mention everything rather than having "hidden" options :).